.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Iceberg Bug report 🐞
+description: Problems, bugs and issues with Apache Iceberg
+labels: ["kind:bug"]
+body:
+  - type: dropdown
+    attributes:
+      label: Apache Iceberg version
+      description: >
+        What Apache Iceberg version are you using?
+      multiple: false
+      options:
+        - "0.13.1 (latest released)"
+        - "0.13.0"
+        - "0.12.1"
+        - "0.12.0"
+        - "0.11.1"
+        - "0.11.0"
+        - "main (development)"
+    validations:
+      required: true
+  - type: textarea
```

Comment:
What about `input` instead of `textarea`? I doubt this needs to be multi-line. We could also have a dropdown if we think that might be easier, although it may have a lot of options, since there is support for quite a few that I can think of:
- Spark
- Flink
- Trino
- Starburst
- Snowflake
- Dremio
- Starrocks
- Doris
- EMR
- Athena
- PrestoDB
- Flink
- Impala
- Hive

Reply:
You got Flink twice ;) I've updated it to a list and also added Other (also made it optional to keep the bar low).